### PR TITLE
Adds videos from dotSwift 2016

### PR DESCRIPTION
**Pull request by dbarden**
At least the ones published so far :)

### CODE REVIEW

File: data/videos.yml [hunk 1]
```diff
@@ -1450,6 +1450,42 @@ dotSwift 2015:
...
+    youtube: QCxkaTj7QJs
+  - language: English
+    speakers: [Chris Eidhof]
+    title: "Tiny Networking"
+    tags: [swift, networking]
+    youtube: ewk-XNzXzAA
+  - language: English
+    speakers: [Ayaka Nonaka]
+    title: "Going Swift and Beyond"
+    tags: [swift]
```
Comment (chriseidhof) [inline data/videos.yml]: another tag might be `coordinators`

File: data/videos.yml [hunk 2]
```diff
@@ -1450,6 +1450,42 @@ dotSwift 2015:
...
+    youtube: fdnpexLkdx4
+  - language: English
+    speakers: [Roy Marmelstein]
+    title: "Localization is hard"
+    tags: [swift, localization]
+    youtube: R9_jFbQsFWA
+  - language: English
+    speakers: [Maxim Zaks]
+    title: "Beyond JSON in Swift"
+    tags: [swift]
```
Comment (chriseidhof) [inline data/videos.yml]: This could be tagged with `flatbuffers`

File: data/videos.yml [hunk 3]
```diff
@@ -1450,6 +1450,42 @@ dotSwift 2015:
...
+    youtube: e8K9F8IVu14
+  - language: English
+    speakers: [TJ Usiyan]
+    title: "Swift [and the] Evolution"
+    tags: [swift]
+    youtube: Gh1RgYz--wg
+  - language: English
+    speakers: [Thomas Visser]
+    title: "The Bright Future of Swift"
+    tags: [swift]
```
Comment (chriseidhof) [inline data/videos.yml]: here we could add `futures`

**Comment by chriseidhof** [comment]
Also, we could add links to the video descriptions (it's not a transcript, but it often does have the slides in it): http://www.thedotpost.com

**Comment by dbarden** [comment]
I added the tags.

I tried to add the links to thedotpost on the description, but it was rendered as plain text. Gonna take a look to see if there's a way for them to be rendered as links tonight or tomorrow.

**Comment by dbarden** [comment]
doh, I missed that :(

Yeah, that worked.

**Comment by chriseidhof** [comment]
No worries. Would you mind rebasing so that we can merge it in? Thanks for all your hard work, and sorry that we've made it complicated. I'll add a note to the readme.

**Comment by dbarden** [comment]
Hopefully it's fine now :)

**Comment by chriseidhof** [comment]
Thank you so much!

**Pull request merged**